webapp-django/crashstats/crashstats/tests/conftest.py

```diff
@@ -86,13 +85,10 @@ def set_product_versions(self, versions):
         self.mock_gvfp.return_value = versions
 
 
-class BaseTestViews(ProductVersionsMixin, DjangoTestCase):
+class SuperSearchFieldsMock:
```

Comment:
I extracted this out as a mixin because it wasn't cleaning itself up correctly and this is more correct.